src/PaymentGateways/Gateways/Stripe/BECSGateway.php

```diff
@@ -19,6 +20,18 @@ class BECSGateway extends PaymentGateway
     use BECSMandateForm;
     use HandlePaymentIntentStatus;
 
+    /** @var array */
+    protected $errors = [];
```

Comment:
@kjohnson this should be `errorMessages` right?

Reply:
@jon correct. I can update the name.

Reply:
@jonwaldstein correct. I can update the name.

Reply:
okay cool